pkg/healthz/periodic_test.go

```diff
@@ -43,6 +50,8 @@ var _ = Describe("Periodic", func() {
 
 		Describe("#Start", func() {
 			It("should start the manager", func() {
+				defer p.Stop()
+
 				p.Start()
```

Comment:
It should be stopped not be fore it is started, isn't it?
```suggestion
				p.Start()
				defer p.Stop()

```